pkg/chartutil/dependencies_test.go

```diff
@@ -274,7 +321,7 @@ func TestProcessDependencyImportValuesForEnabledCharts(t *testing.T) {
 	c := loadChart(t, "testdata/import-values-from-enabled-subchart/parent-chart")
 	nameOverride := "parent-chart-prod"
 
-	if err := processDependencyImportValues(c); err != nil {
+	if err := processDependencyImportValues(c, false); err != nil {
```

Comment:
Why would this be `false`? (Or, at least, I don't understand why we would expect the old merge behavior for this test)

Reply:
I think I missed it before. This particular test produces the same results whether the values is true or false. Updating to test the new behavior.